app/src/main/java/com/medmapper/v33001/room/MedicineRoomDatabase.kt

```diff
@@ -7,13 +7,16 @@ import androidx.room.RoomDatabase
 import com.medmapper.v33001.dao.MedicineDAO
 import com.medmapper.v33001.dto.Medicine
 
+/**
+ *
+ */
 @Database(entities = arrayOf(Medicine::class), version = 1, exportSchema = false)
 public abstract class MedicineRoomDatabase : RoomDatabase() {
 
     abstract fun medicineDao(): MedicineDAO
 
+    //Singleton prevents multiple instances of database opening at the same time.
     companion object {
```

Comment:
Moved comment out of the actual code block and added a blank kdoc section for future documentation. Since it is a public class, there should be some documentation for it. 
https://kotlinlang.org/docs/coding-conventions.html#coding-conventions-for-libraries